gengokit/template/template.go

```diff
@@ -94,7 +94,7 @@ func nameServiceNameCliClientClient_mainGotemplate() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "NAME-service/NAME-cli-client/client_main.gotemplate", size: 2788, mode: os.FileMode(420), modTime: time.Unix(1479253692, 0)}
+	info := bindataFileInfo{name: "NAME-service/NAME-cli-client/client_main.gotemplate", size: 2788, mode: os.FileMode(436), modTime: time.Unix(1464111000, 0)}
```

Comment:
So `0644 (rw-r--r--)` in decimal:  

```
0*8^3 + 6*8^2 + 4*8^1 + 4*8^0 
=
0*512 + 6*64 + 4*8 + 4*1 
=
0 + 384 + 32 + 4
=
420
```

So go-bindata converts my flag to an int, which is `420` which does mean `(rw-r--r--)` still.